web/src/main/java/com/ritense/valtimo/web/autoconfigure/SwaggerAutoConfiguration.java

```diff
@@ -45,10 +39,8 @@
 @Configuration
 @EnableConfigurationProperties(value = {SwaggerProperties.class})
 @ConditionalOnProperty(value = {"valtimo.swagger.enabled"})
-@EnableSwagger2
```

Comment:
Since this class no longer uses swagger I think this class and the security configurer should be renamed as well

Same with the log messages below